desktop/hardened-desktop/enhanced-container-isolation/index.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Enhanced Container Isolation - benefits, why use it, how it differs to Docker rootless, who it is for
+keywords: containers, rootless, security, sysbox, runtime
+title: What is Enhanced Container Isolation?
+---
+
+>Note
+>
+>Enhanced Container Isolation is available to Docker Business customers only. 
+
+Enhanced Container Isolation provides an additional layer of security that uses a variety of advanced techniques to harden container isolation without impacting developer productivity. 
+
+These techniques include:
+- Running all containers unprivileged through the Linux user-namespace
+- Restricting containers from modifying Docker Desktop VM settings
+- Vetting some critical system calls to prevent container escapes, and partially virtualizing portions of `/proc` and `/sys` inside the container for further isolation. 
+
+This is done automatically and with minimal performance impact. 
+
+Enhanced Container Isolation helps ensure a strong container-to-host isolation and locks in any security configurations that have been created, for instance through [Registry Access Management policies](../registry-access-management.md) or with [Admin Controls](../admin-controls/index.md). 
+
+>Note
+>
+> Enhanced Container Isolation is in addition to other security techniques used by Docker. For example, reduced Linux Capabilities, Seccomp, AppArmor.
+
+### Who is it for?
+
+- For organizations that want to prevent container attacks and reduce vulnerabilities.
+- For organizations that want to ensure stronger container isolation that is easy and intuitive to implement on developers' machines.
+
+### What happens when Enhanced Container Isolation is switched on?
+
+When Enhanced Container Isolation is enabled using [Admin Controls](../admin-controls/index.md), the following features are enabled: 
+
+- All user containers are automatically run in Linux User Namespaces which ensures stronger isolation.
+- The root user in the container maps to an unprivileged user at VM level.
+- Users can continue using containers as usual, including bind-mounting host directories, volumes, networking configurations, etc.
+- Privileged containers work, but they are only privileged within the container's Linux User Namespace, not in the Docker Desktop VM.
+- Containers can no longer share namespaces with the Docker Desktop VM. For example, `--network=host`, `--pid=host`.
+- Containers can no longer modify configuration files in the Docker Desktop VM.
+- Containers become harder to breach. For example, sensitive system calls are vetted and portions of `/proc` and `/sys` are emulated.
```

Comment:
Looks like this section is technical enough for these details below, otherwise we need a separate section to cover this stuff as we need to explain why certain workflows won't work anymore for security reasons.

- The docker-engine's configuration file is automatically modified, even if matching clauses are already present, in which case they are overridden. Refer to [Docker Engine Configuration Guide](https://docs.docker.com/config/daemon) for details.
    * The `runtimes` section displays a single runtime: `sysbox-runc`. Notice that while `runc` is still the default runtime, this one is not utilized to launch regular containers.
    * The `entitlements` section is explicitly defined to forbid the use of `--network=host` and `--security-insecure` flags during `docker build` attempts.

Reply:
To be covered in the FAQs/Known issues section @ctalledo?